subsys/bluetooth/audio/ascs.c

```diff
@@ -1378,6 +1378,62 @@ static int ase_config(struct bt_ascs *ascs, struct bt_ascs_ase *ase,
 	return 0;
 }
 
+int bt_audio_unicast_server_config_ase(struct bt_conn *conn, struct bt_audio_stream *stream,
+				       struct bt_codec *codec,
+				       const struct bt_codec_qos_pref *qos_pref)
+{
+	struct bt_ascs *ascs;
+	struct bt_ascs_ase *ase;
+	struct bt_audio_ep *ep;
+
+	CHECKIF(conn == NULL || stream == NULL || codec == NULL || qos_pref == NULL) {
+		LOG_DBG("NULL value(s) supplied)");
+		return -EINVAL;
+	}
+
+	ascs = ascs_get(conn);
+	ep = stream->ep;
+
+	if (stream->ep != NULL) {
+		LOG_DBG("Stream already configured for conn %p", stream->conn);
+		return -EALREADY;
+	}
+
+	for (int i = 1; i <= ASE_COUNT; i++) {
+		ase = ase_find(ascs, i);
+
+		if (ase == NULL) {
+			ase = ase_new(ascs, i);
+			break;
+		}
+	}
```

Comment:
Can you please add a small comment stating why we are doing this? Just a small explanation stating that this is to get a "free" ASE